cabi/src/core.rs

```diff
@@ -111,6 +114,9 @@ impl SymbolicErrorCode {
             ErrorKind::MissingSection(..) => SymbolicErrorCode::MissingSection,
             ErrorKind::BadDwarfData(..) => SymbolicErrorCode::BadDwarfData,
             ErrorKind::MissingDebugInfo(..) => SymbolicErrorCode::MissingDebugInfo,
+            ErrorKind::BadJson(..) => SymbolicErrorCode::BadJson,
+            ErrorKind::BadSourcemap(..) => SymbolicErrorCode::BadSourcemap,
+            ErrorKind::CannotFlattenSourcemap(..) => SymbolicErrorCode::CannotFlattenSourcemap,
             ErrorKind::Io(..) => SymbolicErrorCode::Io,
             ErrorKind::Utf8Error(..) => SymbolicErrorCode::Utf8Error,
             // we don't use _ here but the hidden field on error kind so that
```

Comment:
The match must be exhaustive anyway, so `__Nonexhaustive` can be removed

Reply:
I did not generate `__Nonexhaustive`, it's from error chain. If I remove it it will fail because there it will fail to compile and with `_` we accidentally swallow others.